### DISCUSSION TITLE
Setting BindingContext to ViewModel

### DISCUSSION THREAD

**Discussion by dharamhbtik** [dotnet/maui#21660] [Q&A]
There are a few ways to set the Page Bindinig context to View Model :
1. In the Page constructor, inject the view model or create new instance manually and set the Bindig Context
2. Reference  the ViewModel in to the XAML and then set the binding context
3. Custom Implementation, using ViewModelLocater 

We were using PRISM in Xamarin Forms, which provides a good way of setting binding context while registering the classes/views/models etc during app startup.

Just checking if MAUI have any inbuilt capability to set binding context or we have to use above traditional ways for binding?

**Comment by gautambjain** [comment]
Hi Dharmendra,
This link should help: https://learn.microsoft.com/en-us/dotnet/maui/fundamentals/data-binding/basic-bindings?view=net-maui-8.0

**Reply by dharamhbtik** [reply]
@gautambjain  thanks for your reply but this is not what I am looking for. This link provides just a fundamental of the bindings. What I am looking for is something Like PRISM provides in xamarin forms
